cli/cmd/wallet/cmd/account/list_test.go

```diff
@@ -18,7 +18,7 @@ func TestAccountList(t *testing.T) {
363339333833323631363533393339363633393632333033363632363433313331333636363334333633333636333433313334333533313333333033333332363533333333363133333634333133373335363433393336333633323635363436343636333133363332333133303331363636333636333636333631333236313339363636353634363136343635363433373334363233383330333433373633333536343633363632323361376232323733366336663734323233613232333032323263323236393665363436353738323233613232333032323263323236323635363136333666366535663632366336663633366235663732366636663734323233613232333037383330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330323232633232373336663735373236333635323233613762323236353730366636333638323233613232333132323263323237323666366637343232336132323330373833303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303232376432633232373436313732363736353734323233613762323236353730366636333638323233613232333232323263323237323666366637343232336132323330373833303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303330333033303232376437643764222c226869676865737450726f706f73616c223a22376232323339333533303338333733313338333233393333333736363336333933383332363136353339333936363339363233303336363236343331333133363636333433363333363633343331333433353331333333303333333236353333333336313333363433313337333536343339333633363332363536343634363633313336333233313330333136363633363633363633363133323631333936363635363436313634363536343337333436323338333033343337363333353634363336363232336133323764222c226e6574776f726b223a22373037323631373436353732222c2277616c6c6574223a223762323236393634323233613232333533343332333133383335333533333264363236313332333332643334363636313336326433383337363633333264333336313633333533323332333136343338333133313331323232633232363936653634363537383464363137303730363537323232336137623232333933353330333833373331333833323339333333373636333633393338333236313635333933393636333936323330333636323634333133313336363633343336333336363334333133343335333133333330333333323635333333333631333336343331333733353634333933363336333236353634363436363331333633323331333033313636363336363336363336313332363133393636363536343631363436353634333733343632333833303334333736333335363436333636323233613232333533373331333433363334333333353264333433393336333332643334333236343335326436323635333736353264363636313635363133393338333233303636363333393337323237643263323237343739373036353232336132323438343432323764222c2277616c6c657454797065223a224844227d",
```

Comment:
@olegshmuelov If we save just the slot and not block, why it became bigger?

Reply:
this storage includes slashing protection store with values while previous one has not